unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Passport/Passport/PlayerPassportHUDView.cs

```diff
@@ -0,0 +1,40 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.Events;
+
+public class PlayerPassportHUDView : MonoBehaviour
+{
+    [SerializeField] private PassportPlayerInfoComponentView playerInfoView;
+    [SerializeField] private PassportPlayerPreviewComponentView playerPreviewView;
+    [SerializeField] private PassportNavigationComponentView passportNavigationView;
+
+    public PassportPlayerInfoComponentView PlayerInfoView => playerInfoView;
+    public PassportPlayerPreviewComponentView PlayerPreviewView => playerPreviewView;
+    public PassportNavigationComponentView PassportNavigationView => passportNavigationView;
+
+    [SerializeField] internal Button hideCardButton;
+    [SerializeField] internal GameObject container;
+
+    public static PlayerPassportHUDView CreateView()
+    {
+        return Instantiate(Resources.Load<GameObject>("PlayerPassport")).GetComponent<PlayerPassportHUDView>();
+    }
+
+    public void Initialize(UnityAction cardClosedCallback)
```

Comment:
replace by adding: `public event Action OnCardClosed;`. Initialize method should not be necessary anymore